packages/backend/src/db/storage/constants.ts

```diff
@@ -5,3 +5,26 @@ export const TILES_SAMPLE_URL_DESCRIPTION = 'View a sample table'
 export const USER_EMAIL_PLACEHOLDER = 'user_email'
 export const TILE_ID_PLACEHOLDER = 'tile_table_id'
 export const TILE_COL_DATA_PLACEHOLDER = 'tile_col_data'
```

Comment:
i would suggest just 
```suggestion
export const USER_EMAIL_PLACEHOLDER = '<<user_email>>'
export const TILE_ID_PLACEHOLDER = '<<tile_table_id>>'
export const TILE_COL_DATA_PLACEHOLDER = (colName) => `<<tile_col_data.${colName}>>`
```

calling CREATE_TEMPLATE_PLACEHOLDER(USER_EMAIL_PLACEHOLDER) is too verbose and confusing

Reply:
ok i will simplify this